stylelint.config.js

```diff
@@ -14,5 +14,6 @@ module.exports = {
         "scss/operator-no-newline-after": null,
         "scss/double-slash-comment-whitespace-inside": null,
         "scss/dollar-variable-pattern": null,
+        "declaration-block-single-line-max-declarations": null,
```

Comment:
If we rebase this after merging in the tidying of unnecessary compiled assets then we shouldn't need this line, since it was only relevant for the compiled CSS.

Reply:
location-autocomplete.min.css still triggers this error -- should this file be deleted?

Reply:
It's possible we should tell stylelint to ignore /includes